nodeup/pkg/model/kube_apiserver.go

```diff
@@ -316,19 +316,30 @@ func (b *KubeAPIServerBuilder) buildPod() (*v1.Pod, error) {
 		}
 	}
 
+	var mainCluster, eventsCluster string
```

Comment:
Consider renaming these to `mainEtcdCluster` and `eventsEtcdCluster` for clarifying which kind of cluster we mean.